### ISSUE TITLE
MinerPayouts in Block is incompatible with old block, making SendBloc RPC fail

### ISSUE THREAD

**Issue by GlenDC** [rivine/rivine#305]
```
could not decode type []types.Block: could not decode type []types.CoinOutput: could not decode type []uint8: encoded slice is too large
```

Also: we seem to ignore the error returned by the logic that triggers this error as can be seen at: https://github.com/rivine/rivine/blob/a4266547099c90127d894ed223bc952b9b843a04/modules/consensus/synchronize.go#L547, unless I miss something, that should be `return err` instead of `return nil`, no?

**Comment by GlenDC** [comment]
I don't see any reason to change this property in the Block. Hence I propose the following, we define a new struct `MinerPayout`:

```go
type MinerPayout struct {
     Value Currency `json:"value"`
     UnlockHash UnlockHash `json:"unlockhash"`
}
```

Which we than use for MinerPayouts such that the `Block` struct becomes:

```go
type Block struct {
    ParentID     BlockID                 `json:"parentid"`
    Timestamp    Timestamp               `json:"timestamp"`
    POBSOutput   BlockStakeOutputIndexes `json:"pobsindexes"`
    MinerPayouts []MinerPayout           `json:"minerpayouts"`
    Transactions []Transaction           `json:"transactions"`
}
```

This makes it also that we no longer need the solution implemented as part of #302, and instead can encode those miner payouts directly, once again.

**Comment by GlenDC** [comment]
fixed by 4f17bba
